<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QFRDRImagingFCSMaskByIntensity</class>
 <widget class="QDialog" name="QFRDRImagingFCSMaskByIntensity">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>954</width>
    <height>564</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Mask from Image ...</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,0">
     <item>
      <widget class="QFPlotter" name="pltMain" native="true"/>
     </item>
     <item>
      <layout class="QFormLayout" name="formLayout_2">
       <property name="fieldGrowthPolicy">
        <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Data Range:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="labRange">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Threshold:</string>
         </property>
         <property name="buddy">
          <cstring>edtLevel</cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QFDoubleEdit" name="edtLevel"/>
         </item>
         <item>
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QFDoubleEdit" name="edtLevel_2"/>
         </item>
        </layout>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Threshold 2:</string>
         </property>
         <property name="buddy">
          <cstring>edtLevel</cstring>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QFDoubleEdit" name="edtLevel2"/>
         </item>
         <item>
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QFDoubleEdit" name="edtLevel2_2"/>
         </item>
        </layout>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Mode:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QComboBox" name="cmbMaskingMode">
           <item>
            <property name="text">
             <string>mask out-of-range pixels</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>mask in-range pixels</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>mask smaller threshold</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>mask larger threshold</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>DualView:</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QComboBox" name="cmbDualView">
         <item>
          <property name="text">
           <string>one channel</string>
          </property>
          <property name="icon">
           <iconset resource="qfrdrimagingfcs.qrc">
            <normaloff>:/imaging_fcs/dvnone.png</normaloff>:/imaging_fcs/dvnone.png</iconset>
          </property>
         </item>
         <item>
          <property name="text">
           <string>horizontal</string>
          </property>
          <property name="icon">
           <iconset resource="qfrdrimagingfcs.qrc">
            <normaloff>:/imaging_fcs/dvhor.png</normaloff>:/imaging_fcs/dvhor.png</iconset>
          </property>
         </item>
         <item>
          <property name="text">
           <string>vertical</string>
          </property>
          <property name="icon">
           <iconset resource="qfrdrimagingfcs.qrc">
            <normaloff>:/imaging_fcs/dvver.png</normaloff>:/imaging_fcs/dvver.png</iconset>
          </property>
         </item>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QCheckBox" name="chkEqualChannels">
         <property name="text">
          <string>both channels equal</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="7" column="1">
        <widget class="QPushButton" name="pushButton">
         <property name="text">
          <string>&amp;update mask</string>
         </property>
        </widget>
       </item>
       <item row="8" column="1">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="9" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>mask m&amp;ode:</string>
         </property>
         <property name="buddy">
          <cstring>cmbMaskMode</cstring>
         </property>
        </widget>
       </item>
       <item row="9" column="1">
        <widget class="QComboBox" name="cmbMaskMode">
         <item>
          <property name="text">
           <string>replace</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>add</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>remove</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QFDoubleEdit</class>
   <extends>QLineEdit</extends>
   <header>qfdoubleedit.h</header>
   <slots>
    <signal>valueChanged(double)</signal>
   </slots>
  </customwidget>
  <customwidget>
   <class>QFPlotter</class>
   <extends>QWidget</extends>
   <header>qfplotter.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="qfrdrimagingfcs.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>QFRDRImagingFCSMaskByIntensity</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>266</x>
     <y>554</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>QFRDRImagingFCSMaskByIntensity</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>334</x>
     <y>554</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>edtLevel</sender>
   <signal>editingFinished()</signal>
   <receiver>QFRDRImagingFCSMaskByIntensity</receiver>
   <slot>updateMask()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>784</x>
     <y>50</y>
    </hint>
    <hint type="destinationlabel">
     <x>361</x>
     <y>189</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>edtLevel</sender>
   <signal>returnPressed()</signal>
   <receiver>QFRDRImagingFCSMaskByIntensity</receiver>
   <slot>updateMask()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>784</x>
     <y>50</y>
    </hint>
    <hint type="destinationlabel">
     <x>408</x>
     <y>185</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>QFRDRImagingFCSMaskByIntensity</receiver>
   <slot>updateMask()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>942</x>
     <y>233</y>
    </hint>
    <hint type="destinationlabel">
     <x>397</x>
     <y>224</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>edtLevel</sender>
   <signal>textChanged(QString)</signal>
   <receiver>QFRDRImagingFCSMaskByIntensity</receiver>
   <slot>updateMask()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>784</x>
     <y>50</y>
    </hint>
    <hint type="destinationlabel">
     <x>390</x>
     <y>289</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cmbDualView</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>QFRDRImagingFCSMaskByIntensity</receiver>
   <slot>updateDualView()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>942</x>
     <y>135</y>
    </hint>
    <hint type="destinationlabel">
     <x>613</x>
     <y>115</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkEqualChannels</sender>
   <signal>toggled(bool)</signal>
   <receiver>QFRDRImagingFCSMaskByIntensity</receiver>
   <slot>updateDualView()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>942</x>
     <y>158</y>
    </hint>
    <hint type="destinationlabel">
     <x>621</x>
     <y>106</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>edtLevel2</sender>
   <signal>editingFinished()</signal>
   <receiver>QFRDRImagingFCSMaskByIntensity</receiver>
   <slot>updateMask()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>732</x>
     <y>70</y>
    </hint>
    <hint type="destinationlabel">
     <x>657</x>
     <y>8</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>edtLevel2</sender>
   <signal>returnPressed()</signal>
   <receiver>QFRDRImagingFCSMaskByIntensity</receiver>
   <slot>updateMask()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>698</x>
     <y>66</y>
    </hint>
    <hint type="destinationlabel">
     <x>616</x>
     <y>26</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cmbMaskingMode</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>QFRDRImagingFCSMaskByIntensity</receiver>
   <slot>updateMask()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>808</x>
     <y>97</y>
    </hint>
    <hint type="destinationlabel">
     <x>582</x>
     <y>77</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>edtLevel2_2</sender>
   <signal>valueChanged(double)</signal>
   <receiver>QFRDRImagingFCSMaskByIntensity</receiver>
   <slot>updateMask()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>838</x>
     <y>65</y>
    </hint>
    <hint type="destinationlabel">
     <x>578</x>
     <y>58</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>edtLevel_2</sender>
   <signal>valueChanged(double)</signal>
   <receiver>QFRDRImagingFCSMaskByIntensity</receiver>
   <slot>updateMask()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>829</x>
     <y>46</y>
    </hint>
    <hint type="destinationlabel">
     <x>579</x>
     <y>42</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>updateMask()</slot>
  <slot>updateDualView()</slot>
 </slots>
</ui>
